Dockerfile

```diff
@@ -40,7 +40,8 @@ RUN conda install -n cudf -y -c numba -c conda-forge -c nvidia -c rapidsai -c de
       pyarrow=${PYARROW_VERSION} \
       nvstrings \
       cmake=3.12 \
-      gtest=1.8.0
+      gtest=1.8.0 \
```

Comment:
We have googletest as a submodule and build from source as part of our CMake infrastructure to avoid ABI issues, there shouldn't be a need to install it via conda.

Reply:
Given gtest is already in there will ignore this for now and resolve in a future PR.